app/javascript/controllers/click_toggle_controller.js

```diff
@@ -0,0 +1,24 @@
+import { Controller } from '@hotwired/stimulus'
+
+export default class extends Controller {
+  static classes = [ "flippable" ]
+  static targets = [ "trigger", "destination" ]
+
+  connect () {
+    this.triggerTargets.forEach(triggerElement => {
+
+      triggerElement.addEventListener('click', () => {
+        console.log(`clicked`, this.flippableClass)
+        this.destinationTargets.forEach(element => {
+          element.classList.toggle(this.flippableClass)
+        })
+      })
+    })
+  }
+
+  disconnect () {
+    this.triggerTargets.forEach(triggerElement => {
+      triggerElement.removeEventListener('click')
```

Comment:
This doesn't do anything, you'll [need to pass the function which is registered as the event handler](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener#matching_event_listeners_for_removal).

Reply:
Good catch, fixing!